src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/UpdateAzureIntegrationAccountAssemblyCommand.cs

```diff
@@ -0,0 +1,172 @@
+﻿// ----------------------------------------------------------------------------------
+//
```

Comment:
If this is a Set (PUT), please rename the filename to reflect that.  If this is an Update (PATCH), please update the cmdlet name to reflect that.  Same for the BatchConfiguration cmdlet.

Reply:
This is for Set. Renamed the files